BUILD

```diff
@@ -201,19 +201,23 @@ release_validate_deps(
     tags = ["manual"]  # in order for bazel test //... to not fail
 )
 
-container_image(
+docker_container_image(
     name = "assemble-docker",
     base = "@vaticle_ubuntu_image//image",
     tars = [":assemble-linux-targz"],
     directory = "opt",
     workdir = "/opt/typedb-all-linux",
     ports = ["1729"],
+    env = {
+        "LANG": "C.UTF-8",
+        "LC_ALL": "C.UTF-8",
+    },
```

Comment:
Specify that, when this Docker image is launched, it should be launched with the environment variables `LANG=C.UTF-8` and `LC_ALL=C.UTF-8`.

This sets the system's locale to `C.UTF-8` which can represent all Unicode characters.